src/app/page.js

```diff
@@ -1,61 +1,161 @@
+              <a href="https://github.com/elidakirigo">
+                <img
+                  class="rounded"
+                  src="https://avatars.githubusercontent.com/u/42931101?v=4"
+                ></img>
+                <h1 className="bold p-1 pt-2 text-s text-black center">
+                  Eli da
+                </h1>
               </a>
-              
             </div>
-            <div  className=' bg-white w-1/4 rounded p-2 m-5'>
-              <a href='https://github.com/Silent-Watcher'>
-                <img src='https://avatars.githubusercontent.com/u/91375198?v=4'></img>
-                <h1 className='bold text-black center'>Ali t.nazari</h1>
+            <div className="bg-blue-100 w-1/4 rounded p-2 m-5">
+              <a href="https://github.com/Axorax">
+                <img
+                  class="rounded"
+                  src="https://avatars.githubusercontent.com/u/78349410?v=4"
+                ></img>
+                <h1 className="bold p-1 pt-2 text-s text-black center">
+                  Axorax
+                </h1>
+              </a>
+            </div>
+            <div className="bg-blue-100 w-1/4 rounded p-2 m-5">
+              <a href="https://github.com/Silent-Watcher">
+                <img
+                  class="rounded"
+                  src="https://avatars.githubusercontent.com/u/91375198?v=4"
+                ></img>
+                <h1 className="bold p-1 pt-2 text-s text-black center">
+                  Ali t.nazari
+                </h1>
               </a>
-              
             </div>
-            
           </div>
-          
-          
         </div>
+
+        <footer class="w-[40rem] max-w-full bg-white rounded-lg shadow m-4 dark:bg-gray-800">
+          <div class="w-full mx-auto max-w-screen-xl p-4 md:flex md:items-center md:justify-between">
+            <span class="text-sm text-gray-500 sm:text-center dark:text-gray-400">
+              © 2023{" "}
+              <a
+                href="https://web-dev-tools.vercel.app/"
+                class="hover:underline"
+              >
+                WebDevTools
+              </a>
+              . All Rights Reserved.
+            </span>{" "}
+            &emsp;
+            <ul class="flex flex-wrap items-center mt-3 text-sm font-medium text-gray-500 dark:text-gray-400 sm:mt-0">
+              <li>
+                <a
+                  href="https://web-dev-tools.vercel.app/"
+                  class="mr-4 hover:underline md:mr-6 "
+                >
+                  Home
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://github.com/Bashamega/WebDevTools"
+                  class="hover:underline"
+                >
+                  Repository
+                </a>
+              </li>
+              <li>
+                &emsp;
+                <a href="https://github.com/Axorax" class="hover:underline">
```

Comment:
Dear Axorax. Since this is an open source project, and not a personal project, it is not fair to add your name a after the copy right. If you want to add a name, you can add some thing like Web Dev Tools, and redirect them to the _web dev tools_ repo
```suggestion
                <a href="https://github.com/bashamega/WebDevTools" class="hover:underline">
                Web Dev Tools
                </a>
```